cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/MockLogReader.java

```diff
@@ -77,12 +78,28 @@
   private final DefaultStore store;
   private final List<LogEvent> logEvents = Lists.newArrayList();
   private final Map<ProgramId, RunRecord> runRecordMap = Maps.newHashMap();
+  private int sourceId; 
 
   @Inject
   MockLogReader(DefaultStore store) {
     this.store = store;
   }
 
+  private void setStartAndRunning(final ProgramId id, final String pid, final long startTime) {
```

Comment:
+1 for moving these from the main to the test classes. But is MockLogReader the right place for these methods?